### PR TITLE
Update CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by mackenbach**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -10,3 +10,6 @@
 - Simon Barendse [@simonbarendse](https://github.com/simonbarendse)
 - Valerio Barrila [@ninjatux](https://github.com/ninjatux)
 - Floris van der Grinten [@florisvdg](https://github.com/florisvdg)
+- Márton Soos [@Marton6](https://github.com/marton6)
+- Teddy Filip [@edif2008](https://github.com/edif2008)
```
Comment (SimonBarendse) [inline CONTRIBUTORS.md:14]: Hi @edif2008, you have made it to the contributor list! :smile:



Do you want this to say Teddy or Eduard?

Comment (SimonBarendse) [reply CONTRIBUTORS.md:14]: Feel free to add a commit to this branch if you want it changed

Comment (edif2008) [reply CONTRIBUTORS.md]: Changed to Eduard to be professional. 😄

Comment (mackenbach) [reply CONTRIBUTORS.md]: Then I'd suggest changing your profile to Eduard as well 😉

Comment (edif2008) [reply CONTRIBUTORS.md]: Done.

**Review by jpcoenen — APPROVED** [review]

**Pull request merged**